Automattic/newspack-block-theme
# feat: update input to match newspack components

### All Submissions:






### Changes proposed in this Pull Request:

This is my first try and there might be more to fix. But I want this merged so I can focus on the search function next.

### How to test the changes in this Pull Request:

1. Check the comment form for example
2. Switch to this branch
3. Notice the differences

### Other information:
